src/seml/database.py

```diff
@@ -20,6 +22,19 @@ def get_collection(collection_name, mongodb_config=None, suffix=None):
 
 
 def get_mongo_client(db_name, host, port, username, password, **kwargs):
+    if 'ssh_config' in kwargs:
+        try:
+            from sshtunnel import SSHTunnelForwarder                 
+        except ImportError as e:
+            print('Opening ssh tunnel requires https://sshtunnel.readthedocs.io/en/latest/')
```

Comment:
Use `logging.error` here. Also it may be best to directly provide a pip command `pip install sshtunnel`

Reply:
Isn't it better to refer to the documentation? I do not care too much tbh